src/libraries/Common/src/System/Security/Cryptography/ECDsaCng.ImportExport.cs

```diff
@@ -264,5 +292,48 @@ public override bool TryExportEncryptedPkcs8PrivateKey(
                 destination,
                 out bytesWritten);
         }
+
+        private bool PlaintextOnlyExport
```

Comment:
It's actually EncryptedOnlyExport... Plaintext is what's not allowed :smile:

Reply:
The name makes a certain amount of sense to me, but probably only me. So, will fix it.